packages/@tinacms/graphql/src/database/datalayer.ts

```diff
@@ -214,6 +230,105 @@ export const coerceFilterChainOperands = (
   return result
 }
 
```

Comment:
Extracting these functions out so they can be re-used for both the list true/false cases